docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -18,12 +18,13 @@ IO Managers are user-provided objects that store asset and op outputs and load t
-| ----------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| <PyObject module="dagster" object="io_manager" decorator /> | A decorator used to define IO managers.                                                                                                                                                                     |
-| <PyObject module="dagster" object="IOManager" />            | Base class for user-provided IO managers.                                                                                                                                                                   |
-| <PyObject object="build_input_context"/>                    | Function for directly constructing a <PyObject object="InputContext"/>, to be passed to the <PyObject object="IOManager" method="load_input"/> method. This is designed primarily for testing purposes.     |
-| <PyObject object="build_output_context"/>                   | Function for directly constructing a <PyObject object="OutputContext"/>, to be passed to the <PyObject object="IOManager" method="handle_output"/> method. This is designed primarily for testing purposes. |
+| Name                                                                          | Description                                                                                                                                                                                                 |
+| ----------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject module="dagster" object="ConfigurableIOManager" decorator />        | A base class used to define configurable IO managers which are also resources.                                                                                                                              |
+| <PyObject module="dagster" object="ConfigurableIOManagerFactory" decorator /> | A base class used to specify configuration for more advanced I/O managers.                                                                                                                                  |
+| <PyObject module="dagster" object="IOManager" />                              | Base class for user-provided IO managers.                                                                                                                                                                   |
```

Comment:
i know this is what it said before, but if i were new to IO managers i think this description "Base class for user-provided IOmanagers" meant any io manager i wrote should be off of this class